src/project/qml/MuseScore/Project/SaveToCloudDialog.qml

```diff
@@ -101,6 +112,9 @@ StyledDialogView {
                         Layout.fillWidth: true
                         currentText: root.name
 
+                        navigation.panel: optionsNavPanel
+                        navigation.row: 1
```

Comment:
need to add accessibility
accessible.name = <Save to cloud/musescore.com text> + " " + <Name label text> + " " currentText